src/main/webapp/assets/js/foodMeetings.js

```diff
@@ -94,24 +85,60 @@ $(document).ready(function () {
 		}
 	});
 
-	$("#ImageLinkId").change(function (event) {
-
-		var imageLink = $("#ImageLinkId").val();
-
-		$("#ImageCard").attr("src",imageLink);
-	});
-
 	$("#AddNewMeeting").click(function (event) {
 		
 		$("#NewMeetingCard").show();
-		foodMeetings.masonry({
-			itemSelector: '.grid-item',
-			columnWidth: 50
-		});
+		reorganizeCards();
 	});
 
 	$("#cancelCreateMeeting").click(function (event) {
 		location.reload(); 
 	});
+	
+	var onModalHide = function() {
+		var imageLink = $("#imageLink").val();
+		$("#ImageCard").attr("src", imageLink);
+		reorganizeCards();
+		
+	};
+	
+	$("#ImageCard").click(function () {
+		
 
-});
+		$('#modal1').openModal({
+			complete : onModalHide
+		});
+
+		$('#modal1').load('searchImage',function(data){
+			$("#imageLink").change(function (event) {
```

Comment:
I think this code related to the searchImages modal should go in another script file since we have actually another jsp file for the searchImages. Do you think is possible?